index.html

```diff
@@ -6,6 +6,392 @@
+				<div class="grayC">
+					Certainly!
+				</div>
+			</section>
+			<img src="assets/section5-illo.svg" alt="Two banknotes">			
+		</section>
+
+		<section class="row underFlorence">
+			<section class="firstHalf">
+				<h1>SHE CHEERS YOU ON.</h1>
+				<p>Florence can send you occasional notes of encouragement to keep you motivated.</p>
+				<p>For patients with asymptomatic conditions, this offers a sense of progress.</p>
+			</section>
+			<section class="secondHalf">
+				<div class="grayC">
+					Hey Mike! Nice job, your cholesterol is down 20%.
+				</div>
+				<img src="assets/florence-avatar.svg" alt="Florence's portrait">
+				<div class="grayC">
+					Treat yourself to something nice! <img src="assets/smile.png" alt="A smiling emoji">
+				</div>
+			</section>
+			<img src="assets/section6-illo.svg" alt="Florence saying good job to you">			
+		</section>
+
+		<section class="row underFlorence">
+			<section class="firstHalf">
+				<h1>SHE KEEPS YOU HONEST.</h1>
+				<p>If Florence notices you're not staying on top of your plan, she can message family members or others in your support network.</p>
+			</section>
+			<section class="secondHalf">
+				<img src="assets/florence-avatar.svg" alt="Florence's portrait">
+				<div class="grayC">
+					Hi Kim, this is an automated message from FlorenceApp.io. Your brother Mike hasn't taken his meds in three days. You might want to give him a call. <a href="#">(919)232-0232</a>
+				</div>
+			</section>
+			<img src="assets/section7-illo.svg" alt="A red telephone receiver">			
+		</section>
+	</section>
+	<!-- end meet Florence -->
+	<section class="row">
+		<span class="subTitle">BEHIND THE SCENES</span>
+		<h2 class="sectionDescription">While Florence is only a concept, here are some things that will be helpful in making her come alive.</h2>
+		<section id="florenceBody">
+			<figure>
+				<img src="assets/florence-capeless.svg" alt="Florence without a cape">
+				<figure class="tagLeft" id="knowledge">
+					<img src="assets/icon-knowledge-base.png" alt="A closed book with a medical cross on the cover">
+					<figcaption>
+					<h3>Medical Knowledge Bases</h3>
+					Side effects and drug interactions are currently available in services like <a href="#">Epocrates</a> and <a href="#">Medscape</a>. Though designed for long-form reading, it could be pruned and adapted to fit messaging constraints.  
+					</figcaption>`
+				</figure>
+				<figure class="tagRight" id="content">
+					<img src="assets/icon-comprehension.svg" alt="Two interlocked gears">
+					<figcaption>
+					<h3>Algorithmic Content</h3>
+					Generating believable messages that don’t feel artificial or templated is key to maintaining the illusion. Tools like <a href="#">Automated Insights Wordsmith</a> hint to possibilities where content can be produced on the fly and personalized to each user. 
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="denial">
```

Comment:
I wouldn't use a figure for these because of content/style issue as noted a few times above.